src/Assets.php

```diff
@@ -66,10 +70,12 @@ public static function add_theme_body_class( $classes = array() ) {
 	}
 
 	/**
-	 * These are used by @woocommerce/components and the block library to set up defaults
-	 * based on user-controlled settings from WordPress. Only use this in wp-admin.
+	 * Returns javascript to inject as data for enqueued wc-blocks-settings script.
+	 *
+	 * @return string;
+	 * @since 2.4.0
 	 */
-	public static function print_script_wc_settings() {
+	protected static function wc_settings_data() {
```

Comment:
Personal taste here, but I usually like functions to start with a verb so it's clear what they do. Here, without looking at the method's code, I can't know if it's printing or returning the settings data. How would `get_wc_settings_data` or `get_wc_settings_data_script` sound here?

Same applies to `wc_block_data`.

Reply:
Ya I can add that.  I had in mind some more ideas around improving things here (but didn't want to do too much in one pull).  Still, I'm not sure I'll get around to those ideas soon so I'll make the name change you suggest.

Reply:
see bcdcb55